lib/main.js

```diff
@@ -70,6 +71,9 @@ ava.test.before(async () => {
   console.log(`Downloading device type OS into ${imagePath}`)
   await resinio.downloadDeviceTypeOS(options.deviceType, options.resinOSVersion, imagePath)
 
+  console.log('Enable delta')
```

Comment:
nitty gritty, but should this be `Enabling deltas` to phrase it as the other log comments?

Reply:
oh yeah, changed.